docs/components/modeler/web-modeler/play-your-process.md

```diff
@@ -155,7 +155,8 @@ For more information about terms, refer to our [licensing and terms page](https:
 
 ### Camunda 8 SaaS
 
-In Camunda 8 SaaS, Play is available to all Web Modeler users with editor or admin permissions within a project.
+In Camunda 8 SaaS, Play is available to all Web Modeler users with commenter, editor, or admin permissions within a project.
+Additionally, within their organization, users need to have a [role](/components/console/manage-organization/manage-users.md#roles-and-permissions) which has deployment privileges.
 Enterprise users need an admin to enable Play by opting in to [alpha features](/components/console/manage-organization/enable-alpha-features.md).
```

Comment:
shouldn't we remove this as well? 🤔

Reply:
That would have been my follow-up question, too. There is also the reference to alpha in Self-Managed. I'd rather create a ticket to clean this up on one swoop, but I can also remove the line here. wdyt?

Reply:
whatever you prefer :)

Reply:
Seperat ticket ;)